instantsearch/src/main/java/com/algolia/instantsearch/transformer/SearchResultsHandler.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *  * Copyright (c) 2018 Algolia
+ *  * http://www.algolia.com/
+ *  *
+ *  * Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  * of this software and associated documentation files (the "Software"), to deal
+ *  * in the Software without restriction, including without limitation the rights
+ *  * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  * copies of the Software, and to permit persons to whom the Software is
+ *  * furnished to do so, subject to the following conditions:
+ *  *
+ *  * The above copyright notice and this permission notice shall be included in
+ *  * all copies or substantial portions of the Software.
+ *  *
+ *  * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  * THE SOFTWARE.
+ *
+ */
+
+package com.algolia.instantsearch.transformer;
+
+/**
+ * Handles completion of a custom back-end API request.
+ */
+public interface SearchResultsHandler<Results> {
+
+    /***
+     * Called when the request has completed, either successfully or failing.
+     * <p>
+     * NOTE: One and only one of either <code>content</code> or <code>error</code> is guaranteed to be not null.
+     * </p>
+     *
+     * @param content
+     * @param error
```

Comment:
would add a succinct description of each params to improve DX

Reply:
There are several other cases of undocumented parameters, I won't comment those but the same applies 🙃